### PR TITLE
V0.8.0 rc

### PR DESCRIPTION
**Pull request by rhornung67**
# Summary

- This PR is to move all work on develop to main for v0.8.0 release. 
- Since almost all changes have been approved and merged into develop, please approve this PR since it will be merged, but review the other PR that contains only changes to develop: https://github.com/LLNL/axom/pull/1146

### CODE REVIEW

**Review by white238 — APPROVED** [review]

File: src/cmake/AxomVersion.cmake [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2017-2022, Lawrence Livermore National Security, LLC and
```
Comment (white238) [inline src/cmake/AxomVersion.cmake:1]: There should be a version change in this file

**Comment by rhornung67** [comment]
Folks, can we get some eyeballs on this so we can put out the release today or tomorrow? Multiple codes are asking for it.

**Review by bmhan12 — APPROVED** [review]
@BradWhitlock should be added to the `.mailmap` and `CITATION.cff` files

**Comment by rhornung67** [comment]
> @BradWhitlock should be added to the `.mailmap` and `CITATION.cff` files

Done.

**Comment by rhornung67** [comment]
To be honest, I don't understand how the various entries in the mailmap file get generated. It looks like a combination of manual entry and automation.

**Review by agcapps — APPROVED** [review]

**Comment by white238** [comment]
> To be honest, I don't understand how the various entries in the mailmap file get generated. It looks like a combination of manual entry and automation.

It's me. I periodically look at our git log of authors and add new entries to the mailmap file to combine them.

**Comment by rhornung67** [comment]
So a lot of the entries reference accounts on old LC machines that no longer exist (e.g., cab). Should those be culled out?

**Comment by white238** [comment]
> So a lot of the entries reference accounts on old LC machines that no longer exist (e.g., cab). Should those be culled out?

No, these are entries from the git author log that need to be merged to the real email of the person. The left side is the correct one, the right is the extra email or name that isnt quite right but has a commit with that author/email combo.

**Comment by rhornung67** [comment]
I'll merge this and push out the release tomorrow morning. If anyone wants to take a look at it, please do it before tomorrow.

**Pull request merged**